MonoGame.Framework/Utilities/BinaryReaderEx.cs

```diff
@@ -10,7 +10,7 @@ namespace MonoGame.Framework.Utilities
     public class BinaryReaderEx : BinaryReader
     {
         public BinaryReaderEx(Stream input) : base(input)
-        {
+        {            
```

Comment:
We have no need for the changes to BinaryReaderEx / BinaryWritterEx.

Reply:
I'll have to see how to undo all the changes from the backwards compatibility version.